src/utils.ts

```diff
@@ -121,3 +121,11 @@ export abstract class BaseCLICommand extends Command {
     return Promise.resolve()
   }
 }
+
+export function setDifference<T> (setA: Set<T>, setB: Set<T>): Set<T> {
```

Comment:
Isn’t this rather `setExtent`?

Reply:
No. This is copy&pasted from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set 😅